eras/conway/impl/cddl-files/extra.cddl

```diff
@@ -11,11 +11,20 @@ nonempty_oset<a> = #6.258([+ a]) / [+ a]
 positive_int = 1 .. 18446744073709551615
 
 unit_interval = #6.30([1, 2])
```

Comment:
I'm thinking that it might be a good idea to have `#6.30([1, 1])` and  `#6.30([0, 1])` also as possibilities, that way we can test more edge cases.

Reply:
It will be better to create a PR that splits out the concerns of testing and specifying.

One that will be used for testing will contain a whole bunch of random values, while the one o that used for specifying will have the correct types.

We can do it at some later point. I only created this PR because we were asked to clarify the comment on the `unit_interval`

Reply:
I think it would be a good idea to extend CDDL so that it's possible to express these kinds of constraints. Maybe once `Cuddle` is in a more developed state we can think about that.

Reply:
I think [Cue](https://cuelang.org/) has some interesting ideas that we could borrow.

Reply:
I created a ticket that captures this as a future task https://github.com/input-output-hk/cardano-ledger/issues/3931